### ISSUE TITLE
Rendering of Plotly FigureWidget failed in JupyterLAb

### ISSUE THREAD

**Issue by sim-san** [ml-tooling/ml-workspace#32]
**Describe the bug:**
I write some libary which uses the Plotly FigureWidget to create some interactive plot. This plot runs in the normal jupyter notebook. But in the jupyter lab I only see the message:  "loading widget..."

**Expected behaviour:**
TODO

**Steps to reproduce the issue:**
TODO

**Technical details:**

- Workspace version: 0.8.7
- Docker version: Docker version 19.03.5, build 633a0ea838
- Host Machine OS (Windows/Linux/Mac): Linux
- Command used to start the workspace: ml-hub
- Browser (Chrome/Firefox/Safari): Firefox

**Possible Fix:**

While searching for the Jupyter-Lab extension I saw that the @jupyterlab/plotly extension is deprecated.

> This package is being deprecated. Please use the Plotly-supported jupyterlab-plotly (https://www.npmjs.com/package/jupyterlab-plotly). See the plotly.py README for more info: https://github.com/plotly/plotly.py#jupyterlab-support-python-35)

**Additional context:**
Installation in the Dockerfile: https://github.com/ml-tooling/ml-workspace/blob/ae85a0261322da44ec70ad15b1ad69adfd7e5f55/Dockerfile#L736

**Comment by sim-san** [comment]
**Steps to reproduce the issue:**
Run this cell inside the jupyter lab
```
import plotly.graph_objects as go

f = go.FigureWidget()
f.add_scatter(y=[2, 1, 4, 3]);
f.add_bar(y=[1, 4, 3, 2]);
f.layout.title = 'Hello FigureWidget'
f.show()
```

**Comment by sim-san** [comment]
A normal figure works:
```
import plotly.graph_objects as go
fig = go.Figure()
fig.add_trace(go.Scatter(y=[2, 1, 4, 3]))
fig.add_trace(go.Bar(y=[1, 4, 3, 2]))
fig.update_layout(title = 'Hello Figure')
fig.show()
```

**Comment by lukasmasuch** [comment]
Thanks for reporting. The issue seems to be fixed with the newest version (`0.10.4`).
